harness/determined/common/util.py

```diff
@@ -211,8 +211,7 @@ def parse_protobuf_timestamp(ts: str) -> datetime.datetime:
 
 
 def is_protobuf_timestamp(ts: str) -> bool:
-    """
-    Validates that a string timestamp is in a Protobuf-compatible format by attempting to
+    """Validates that a string timestamp is in a Protobuf-compatible format by attempting to
```

Comment:
nit: this docstring doesn't conform to PEP-8. Summaries need to fit on one line.

I think you can drop the "how" here to get it to fit. Possibly interesting to a caller, but not so interesting it needs to be in the "what the thing does" summary.

Reply:
good point. done.